### PR TITLE
added database migration feature and scripts

### PR DESCRIPTION
**Pull request by ruchira-wyncore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+
+###How to perform database migration
+1) Fork and clone this repository and go under resource/db/migration.Ignore this if you have done this already.
+
+2) Under this you have to create your migration script which is basically  a file containing sql commands which you want to execute.The .sql file naming conventions follows a pattern as specified by flywaydb.For this repository I have used the pattern like "V1.1__CREATE_REPORT_MASTER"  where V stands for the version name and sequencing starts from 1.1, 1.2 and so on followed by double underscore and then the file name.Please follow the naming convention else database migration script validation will fail.
+
+3)Make sure your local mysql instance(server) is up.
+
+4)create database wyncore in mysql;Ignore this if it already exists;
```
Comment (srajasekaranwyncore) [inline README.md:14]: Handle the case if the database does not exist. In that case create a new database and then run create table scripts.

**Review by srajasekaranwyncore — COMMENTED** [review]
Completed Review

File: README.md (hunk 1 above)
Comment (ruchira-wyncore) [reply README.md:14]: sure will do that

Comment (ruchira-wyncore) [reply README.md:14]: > Handle the case if the database does not exist. In that case create a new database and then run create table scripts.

I thought of doing that and once I did it it did not work.Realized that in order to establish a jdbc connection from any java app , it assumes that database is already there.Hence this step needs to be manual.

In application.properties:-

## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
spring.datasource.url = jdbc:mysql://localhost:3306/wyncore?useSSL=false

**Review by srajasekaranwyncore — APPROVED** [review]

**Comment by srajasekaranwyncore** [comment]
Done

**Pull request merged**